app/uploaders/disability_claim_document_uploader.rb

```diff
@@ -34,12 +34,12 @@ def set_storage_options!
     if ENV['EVSS_S3_UPLOADS'] == 'true'
       self.fog_credentials = {
         provider:              'AWS',
-        aws_access_key_id:     EVSS_AWS_ACCESS_CREDS[:aws_access_key_id],
-        aws_secret_access_key: EVSS_AWS_ACCESS_CREDS[:aws_secret_access_key],
-        region:                ENV['EVSS_S3_REGION']
+        aws_access_key_id:     ENV['EVSS_AWS_ACCESS_KEY_ID'],
+        aws_secret_access_key: ENV['EVSS_AWS_SECRET_ACCESS_KEY'],
+        region:                ENV['EVSS_AWS_S3_REGION']
       }
```

Comment:
This is providing configuration values to Carrierwave so I don't think we want to impose any restrictions on the hash or add overhead by providing indifferent access when it only needs/expects symbolized keys